src/operator/dropout-inl.h

```diff
@@ -35,13 +35,14 @@ enum DropoutOpForwardResource {kRandom};
 namespace mxnet {
 namespace op {
 
-#if defined(USE_STATIC_MKL) && defined(_OPENMP)
+#if defined(USE_MKL) && defined(_OPENMP)
+const int nthr = omp_get_max_threads();
+std::vector<unsigned int> bernoulli_seed(nthr, 1);
```

Comment:
is there a random function that doesn't need seed as input?
We don't require determinism for layers